workflows/format.yml

```diff
@@ -0,0 +1,22 @@
+name: Python Linting and Type Checks
```

Comment:
Shouldn't the workflows be under .github/workflows - https://docs.github.com/en/actions/about-github-actions/understanding-github-actions#workflows?

Reply:
Good catch! This got copied incorrectly, fixed